util/containers/syncmap.go

```diff
@@ -12,7 +12,12 @@ func (m *SyncMap[K, V]) Load(key K) (V, bool) {
 		var empty V
 		return empty, false
 	}
-	return val.(V), true
+	vVal, ok := val.(V)
+	if !ok {
+		var empty V
+		return empty, false
```

Comment:
We should panic here (and in Keys) if there's an unknown type detected, as it means the SyncMap is invalid.

Reply:
Done.

Reply:
Picky ... picky. Actually done now.